index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome cdn link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Africa Peace concert</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <div class="first-navigation">
+        <a href="#"
+          ><ion-icon class="first-nav-icons" name="logo-facebook"></ion-icon
+        ></a>
+        <a href="#"
+          ><ion-icon class="first-nav-icons" name="logo-twitter"></ion-icon
+        ></a>
+        <a href=""><p>English</p></a>
+        <a href=""><p>My Page</p></a>
+        <a href=""><p>Logout</p></a>
+      </div>
+      <div class="second-navigation">
+        <a href="index.html"
+          ><img
+            src="images/africa.png"
+            class="second-nav-img"
+            alt="africa logo"
+            width="50"
+            height="50"
+        /></a>
+        <ul class="desktop-nav">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li>
+            <a class="campaign-p" href="#">PC Africa</a>
+          </li>
+        </ul>

```

Comment:
- [ ] I would suggest you kindly use a `<nav>` tag as the container tag holding your nav menu for proper structuring of the HTML semantics 👍 👍 

> N.B: The HTML uses semantic tags, sections use `section`, articles use `article`, and the nav and footer use `nav` and `footer`.

Reply:
noted.